cypress/support/pages/homeСatalogue.pageObject.js

```diff
@@ -1,21 +1,52 @@
 import PageObject from '../PageObject';
 
+
 class HomeAndCataloguePageObject extends PageObject {
   url = '/index.html';
 
-  clickOnLink(linkName) {
-    cy.contains('.nav-link', linkName)
-      .click();
+  get laptopBtn() {
+    return cy.get("#itemc");
+  }
+  clickLaptopBtn() {
+    this.laptopBtn.click();
+  }
+  get laptopNameBtn() {
+    return cy.contains(".hrefch", "Sony vanio i7");
+  }
+  clickLaptopNameBtn() {
+    this.laptopNameBtn.click();
+  }
+  get addToCartBtn() {
+    return cy.contains(".btn", "Add to cart");
+  }
+  clickAddToCartBtn() {
+    this.addToCartBtn.click();
+  }
+  get successfulWindow() {
+    cy.on("window:alert", (str) => {
+      expect(str).to.equal("Product added.");
+    });
+  }
+  
+  get window_ok_btn() {
+    return cy.get("button", "OK");
+  }
+  click_window_ok_btn() {
```

Comment:
Function name 'click_window_ok_btn' is not following camelCase naming convention. It should be corrected to 'clickWindowOkBtn'.